lib/model/CIndividualModel.cc

```diff
@@ -70,10 +70,6 @@ const std::string FIRST_BUCKET_TIME_TAG("c");
 const std::string LAST_BUCKET_TIME_TAG("d");
 const std::string FEATURE_MODELS_TAG("e");
 const std::string FEATURE_CORRELATE_MODELS_TAG("f");
-// Extra data tag deprecated at model version 34
-// TODO remove on next version bump
-//const std::string EXTRA_DATA_TAG("g");
-//const std::string INTERIM_BUCKET_CORRECTOR_TAG("h");
```

Comment:
`INTERIM_BUCKET_CORRECTOR_TAG` was only removed in 6.4 (in #97), so we might still see states that contain it (and 6.4+ will silently ignore it).  I think it would be best to leave it commented out until that possibility has gone.

`EXTRA_DATA_TAG` is ancient and can go.  (The comment above it just relates to `EXTRA_DATA_TAG`.)

Reply:
See [c2bef08](https://github.com/elastic/ml-cpp/pull/2267/commits/c2bef089579b57eb3dd64d7d0b8ca2557c43ad7b ).